core/components/atoms/button/Button.tsx

```diff
@@ -20,6 +20,9 @@ export interface ButtonProps extends BaseProps, BaseHtmlProps<HTMLButtonElement>
   size?: Size;
   /**
    * Color of the `Button`
+   *
+   * ** `'success'` will be deprecated soon**
```

Comment:
This message needs to be changed to ** `'success'` has been deprecated and all success button types will now be changed to primary button automatically**